scripts/boxgame.js

```diff
@@ -0,0 +1,119 @@
+class BoxGameComponent {
+    box = {};
+    score = document.createElement("div");
+    squares = [];
+    container = document.createElement('div');
+    gameAreaEl = document.createElement("div");
+    gamebox = {
+        x: Math.floor(800 / 100),
+        y: Math.floor(200 / 100)
+    };
+    player = {
+        speed: 100,
+        square: 1,
+        score: 0
+    };
```

Comment:
the speed element, like the x/y values above, are too tightly coupled to the css style .gameArea